inbm/dispatcher-agent/dispatcher/fota/installer.py

```diff
@@ -78,7 +78,7 @@ def get_product_params(self, platform_product: str) -> Dict:
                 if ' ' in tool_name or tool_name.isspace():
                     raise FotaError(f"FOTA tool name cannot contain spaces - {tool_name}")
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Robustness$</h2>**
**Suggestion:** Consider adding a fallback or default value for tool_name before performing operations on it. This ensures that the code does not raise an exception if tool_name is not found in the parameters.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                tool_name = params.get('firmware_tool', '')
                if ' ' in tool_name or tool_name.isspace():
                    raise FotaError(f"FOTA tool name cannot contain spaces - {tool_name}")
```